compiler/tests/org/jetbrains/kotlin/codegen/debugInformation/IrSteppingTestGenerated.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.codegen.debugInformation;
+
+import com.intellij.testFramework.TestDataPath;
+import org.jetbrains.kotlin.test.KotlinTestUtils;
+import org.jetbrains.kotlin.test.TargetBackend;
+import org.jetbrains.kotlin.test.TestMetadata;
+import org.junit.runner.RunWith;
+import org.junit.runners.BlockJUnit4ClassRunner;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.regex.Pattern;
+
+/** This class is generated by {@link org.jetbrains.kotlin.generators.tests.TestsPackage}. DO NOT MODIFY MANUALLY */
+@SuppressWarnings("all")
+@TestMetadata("compiler/testData/debug/stepping")
+@TestDataPath("$PROJECT_ROOT")
+@RunWith(BlockJUnit4ClassRunner.class)
+public class IrSteppingTestGenerated extends AbstractIrSteppingTest {
+    private void runTest(String testDataFilePath) throws Exception {
```

Comment:
Please add test in 'stepping' subfolder to be sure that nested suites will be generated/executed correctly (if not it could be postponed for separate pr)

Reply:
Can you elaborate on this? Does that mean to put the abstract test suite in /debugInformation/stepping/ folder, like /debugInformation/stepping/AbstractSteppingTest, and generate to see if generator can handle that case?

Reply:
Oh no, i meant just another test data file under ```testData/debug/stepping/smth/sometest.kt```
Actually a new subsuite is generated for each folder under main one

Reply:
Verified locally, such behavior is working. But at this moment I can not think up of test cases suits for putting in a subfolder, maybe I can add such nested test suite later in the future.